pkg/apis/serving/v1alpha1/servingruntime_types.go

```diff
@@ -145,7 +145,8 @@ type ServingRuntimeSpec struct {
 	HTTPDataEndpoint *string `json:"httpDataEndpoint,omitempty"`
 
 	// Configure the number of replicas in the Deployment generated by this ServingRuntime
-	// If specified, this overrides the podsPerRuntime configuration value
+	// If specified, this overrides the podsPerRuntime configuration value.
+	// Note: This configuration is ModelMesh-specific and does not have any effect for non-ModelMesh InferenceServices.
```

Comment:
This means that this field is only read by the modelmesh-controller?

Reply:
Yes.